chain/store/store.go

```diff
@@ -1203,6 +1207,13 @@ func (cs *ChainStore) Export(ctx context.Context, ts *types.TipSet, inclRecentRo
 			return xerrors.Errorf("unmarshaling block header (cid=%s): %w", blk, err)
 		}
 
+		if currentMinHeight > b.Height {
+			currentMinHeight = b.Height
+			if currentMinHeight%builtin.EpochsInDay == 0 {
+				log.Infow("export", "height", currentMinHeight)
+			}
```

Comment:
Might not be the best check as it can easily be skipped due to null rounds but it's really just here to give a basic idea and I doubt it would get skipped frequently enough to really need something better, but open to suggestions.